rtv/subreddit.py

```diff
@@ -89,6 +91,41 @@ def open_link(self):
         global opened_links
         opened_links.add(url)
 
+    @SubredditController.register('p')
+    def post_submission(self):
+        # Abort if user isn't logged in
+        if not self.reddit.is_logged_in():
+            show_notification(self.stdscr, ["Login to reply"])
+            return
+
+        name = self.content.name
+        if '+' in name:
```

Comment:
I think that there are other cases where this will fail as well, eg ``/r/all``. You could do 
```python
sub = self.reddit.get_subreddit(name)
```
to check that the subreddit is valid. It's an extra API call but I think it would be worth it. Also you wouldn't have to guess about the name because you could pass in the subreddit object.
```python
self.reddit.submit(sub, title, text=content)
```